src/templates/drawerDeprecated.js

```diff
@@ -459,8 +478,8 @@ class DrawerDeprecated extends Component {
         const { onCloseComplete, position, onClickOutside } = this.props;
         const animationEvent = this._transitionProps(this.drawerContainerRef);
         const body = document.body;
-        const drawerLength = document.querySelectorAll('.ui.drawer').length;
-        const drawerDeprecatedLength = document.querySelectorAll('.ui.drawer.deprecated').length;
+        const drawerLength = document.querySelectorAll('.ui.drawer-is_modal').length;
+        const drawerDeprecatedLength = document.querySelectorAll('.ui.drawer.deprecated.drawer-is_modal').length;
```

Comment:
And a similar rename here too please.  Feel free to push back and tell me you don't want to rename any of them! :-)

Reply:
Renamed; resolving.